extensions/stackdriver/config/stackdriver_plugin_config.proto

```diff
@@ -29,7 +29,7 @@ message PluginConfig {
   ReporterKind kind = 1;
 
   // Controls whether to export access log.
-  bool export_access_log = 2;
+  bool disable_access_logging = 2;
```

Comment:
This break the existing code. Usually it's forbidden but we are at the very early phase. WDYT @mandarjog

Reply:
Recommendation: add v1alpha1 as a last part in the proto package

Reply:
It is fine to update proto definition at this point, but anyway I versioned the proto. PTAL. Thanks!